README.md

```diff
@@ -0,0 +1,406 @@
+        
+        // Continue using the PN532 from here
+    });
+
+});
+```
+
+## enablePowerSaveMode(*shouldEnable [, callback]*)
+
+Enables or disables a power-saving mode on the PN532.  This mode will apply to all future commands on the PN532 until it is disabled with another call to this method.  Use of power-save mode will add a 1 ms latency to all commands sent, but significantly decreases power consumption.
+
+Note that even if this is set, power save mode will not be entered after certain commands (such as [pollNearbyTags(*tagType, pollAttempts, pollPeriod, callback*)](#pollnearbytagstagtype-pollattempts-pollperiod-callback)) that require state to be stored on the PN532.  This is automatically handled by the class and power save mode will be re-entered when a compatible command is run.
+
+The *callback* must take the following parameters:
+
+- *error*: A string that is null on success.
+- *wasEnabled*: A boolean that matches *shouldEnable*.
+
+### Usage
+
+```squirrel
+function powerSaveCallback(error, succeeded) {
+        // Continue using PN532 with lower power usage 
+}
+
+reader.enablePowerSaveMode(true, powerSaveCallback);
+```
+
+## getFirmwareVersion(*callback*)
+
+Queries the PN532 for its internal firmware version number.
+
+Takes a *callback* with two parameters:
+
+- *error*: A string that is null on success.
+- *version*: A table that contains fields for *IC*, *Ver*, *Rev*, and *Support* as documented in the `GetFirmwareVersion` command from the [PN532 datasheet](http://www.nxp.com/documents/user_manual/141520.pdf).
+
+### Usage
+
+```squirrel
+function firmwareVersionCallback(error, version) {
+        server.log(format("Firmware version: %X:%X.%X-%X", version, version.ver, version.rev, version.support));
+}
+
+reader.getFirmwareVersion(firmwareVersionCallback);
+```
+
+## pollNearbyTags(*tagType, pollAttempts, pollPeriod, callback*)
+
+Repeatedly searches for nearby NFC tags of type *tagType* and passes scan results to the *callback* upon completion.
+
+### Parameters
+
+- *tagType*: An integer representing the baud rate and initialization protocol used during the scan.  It must be taken from the following static class members:
+
+ - `PN532.TAG_TYPE_106_A`: 106 kbps ISO/IEC14443 Type A
+ - `PN532.TAG_TYPE_212`: Generic 212 kbps
+ - `PN532.TAG_TYPE_424`: Generic 424 kbps
+ - `PN532.TAG_TYPE_106_B`: 106 kbps ISO/IEC14443-3B
+ - `PN532.TAG_TYPE_106_JEWEL`: 106 kbps Innovision Jewel
+
+    Any of the above members can also be combined with the flag `PN532.TAG_FLAG_MIFARE_FELICA` where appropriate to specify that only cards with MIFARE or FeliCa support should be polled.
+- *pollAttempts*: An integer representing how many times the PN532 should search for the specified card type. Any value between 0x01 and 0xFE will initiate the corresponding number of polls.  The value 0xFF will poll forever until a card is found.
+- *pollPeriod*: An integer controlling the time in between poll attempts.  It indicates units of 150 ms.
```

Comment:
Is it possible to give the period in seconds and latch to the nearest available multiple of 0.150?